### DISCUSSION TITLE
运行lotus-miner run的时候报错

### DISCUSSION THREAD

**Discussion by ymyjohnny** [filecoin-project/community-china#63] [新手上路]
ERROR: creating node: starting node: could not build arguments for function "github.com/filecoin-project/lotus/node/modules".GetParams (/root/lotus/node/modules/storageminer.go:89): failed to build abi.RegisteredSealProof: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to build dtypes.MinerAddress: received non-nil error from function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): datastore: key not found

有遇到过吗？init已经完成了，运行run报错

**Comment by Imccccc** [comment]
key not found, 确定init参数配置是对的？

**Reply by Sakuya-CN** [reply]
datastore: key not found 导入key了？

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308,另接代封装业务，矿池搭建和运维服务，价格美丽，提供教学指导
